arches/app/src/components/ControlledListManager/editor/ValueEditor.vue

```diff
@@ -0,0 +1,412 @@
+                                onUpdated: focusInput,
+                            },
+                        }"
+                    />
+                </template>
+                <template #body="slotProps">
+                    <span
+                        class="full-width-pointer"
+                        @click.stop="
+                            makeValueEditable(slotProps.data, slotProps.index)
+                        "
+                    >
+                        {{ slotProps.data.value }}
+                    </span>
+                </template>
+            </Column>
+            <Column
+                field="language_id"
+                :header="languageHeader"
+                style="width: 10%; min-width: 8rem; height: 5rem"
+            >
+                <template #editor="{ data, field }">
+                    <Dropdown
+                        v-model="data[field]"
+                        :options="arches.languages"
+                        :option-label="(lang) => `${lang.name} (${lang.code})`"
+                        option-value="code"
+                        :pt="{
+                            input: {
+                                style: {
+                                    fontFamily: 'inherit',
+                                    fontSize: 'small',
+                                },
+                            },
+                            panel: { style: { fontSize: 'small' } },
+                        }"
+                    />
+                </template>
+                <template #body="slotProps">
+                    {{
+                        `${languageNameFromCode(slotProps.data.language_id)} (${slotProps.data.language_id})`
+                    }}
+                </template>
+            </Column>
+            <Column
+                :row-editor="true"
+                style="width: 5%; min-width: 6rem; text-align: center"
+            />
+            <Column style="width: 5%; text-align: center">
+                <template #body="slotProps">
+                    <i
+                        class="fa fa-trash"
+                        role="button"
+                        tabindex="0"
+                        :aria-label="$gettext('Delete')"
+                        @click="issueDeleteValue(slotProps.data)"
+                        @key.enter="issueDeleteValue(slotProps.data)"
+                    />
+                </template>
+            </Column>
+        </DataTable>
+        <AddValue
+            :value-type
+            :make-value-editable
+        />
+    </div>
+</template>
+
+<style scoped>
+.value-editor-container {
+    margin: 1rem 1rem 3rem 1rem;
+    width: 100%;
+}
+
+h4 {
+    color: v-bind(ARCHES_CHROME_BLUE);
+    margin-top: 0;
+    font-size: 1.33rem;
+}
+
+p {
+    font-weight: normal;
+    margin-top: 0;
+    font-size: 1.2rem;
+}
+
+.full-width-pointer {
+    cursor: pointer;
+    display: flex;
+    width: 100%;
```

Comment:
I applied this style to this class to ensure that multi-line notes render properly (read-only) in their cell, but does it make sense semantically or should we use another class?

```suggestion
    width: 100%;
    white-space: pre-wrap;
```

Reply:
Thanks. I'll add it inline in the component for now, and we can have a look at what makes the most sense in #11050. Also will add this on the image metadata table.